src/eval-frame/components/eval-container.jsx

```diff
@@ -14,10 +15,24 @@ export class EvalContainerUnconnected extends React.Component {
   static propTypes = {
     reportOnly: PropTypes.bool.isRequired,
   }
+  state = {
+    hasError: false,
+    error: null,
+    errorInfo: null,
```

Comment:
Don't think there's any need to set error/errorInfo to null here.

Reply:
what should I do here because I need to pass the error and errorInfo to the ErrorPane component.can you help How I should do it.

Reply:
These object keys will have the value of `undefined` if they are unset, which should be enough to present the right information inside the ErrorPane component.